packages/usa-range/src/usa-range.json

```diff
@@ -0,0 +1,4 @@
+{
+  "disabled": false,
+  "aria_disabled": false
+}
```

Comment:
@mejiaj Did you want to remove the json files that had only disabled info?

Reply:
This example is resolved in [50f287c](https://github.com/uswds/uswds/pull/5063/commits/50f287c95f7a67eaa014099df3d2e27682721456)

Happy to move all disabled options into booleans! We'll discuss if disabled variants should also have their own stories as well